app/assets/javascripts/two-pane.js

```diff
@@ -7,7 +7,7 @@ $.fn.tab = function ( option ) {
   }
 }
 
-$(document).on('click', ".table-two-pane td a[href$='edit']", function(e) {
+$(document).on('click', ".table-two-pane td[class$='display-two-pane'] a", function(e) {
```

Comment:
I don't think it's a real pain, because this is an "on click" handler, therefore we are not filtering many elements. Just one or few per page.